### PR TITLE
Add parser for /var/log/rhn/rhn_server_satellite.log

### PR DESCRIPTION
**Pull request by sagaraivale**

### CODE REVIEW

**Comment by sagaraivale** [comment]
@vishwanathjadhav Plz review

**Comment by vishwanathjadhav** [comment]
1. Please add description why you want to add this parser.

**Comment by vishwanathjadhav** [comment]
Log contains should be proper.

File: insights/parsers/rhn_logs.py [hunk 1]
```diff
@@ -171,3 +171,32 @@ class SearchDaemonLog(LogFileOutput):
...
     """
     time_format = '%Y/%m/%d %H:%M:%S'
+
+
+@parser('rhn_server_satellite.log')
+class SatelliteServerLog(LogFileOutput):
+    """
+    Class for parsing the ``var/log/rhn/rhn_server_satellite.log`` file
+
+    Sample log contents::
```
Comment (vishwanathjadhav) [inline insights/parsers/rhn_logs.py]: Please put it in proper format, need to add one blank line after "Sample log contents", please check the html docs before adding blank line, after adding blank line do "make html_debug", check same html doc.

**Comment by xiangce** [comment]
@sagaraivale - Hi, since the PR has already been approved, please re-submit it to the master branch then I will merge them. Thanks.

**Pull request merged**